src/components/SidebarComponents/Options.vue

```diff
@@ -1,55 +1,73 @@
 <script setup lang="ts">
 import { ref, watch } from 'vue'
-import { FillerAlbum, GlobalChartState } from '../../shared'
+import { FillerAlbum, GlobalChartState, GlobalSiteOptions } from '../../shared'
+import { SaveSiteOptions } from '../../storage'
 
 const showOptions = ref(true)
 
 watch(
-	() => GlobalChartState.value.options.chartSize,
+	() => GlobalSiteOptions.value.numberOfSearchResults,
 	() => {
-		let difference =
-			GlobalChartState.value.options.chartSize.columns *
-				GlobalChartState.value.options.chartSize.rows -
-			GlobalChartState.value.chartTiles.length
-
-		if (difference > 0) {
-			while (difference !== 0) {
-				GlobalChartState.value.chartTiles.push(FillerAlbum)
-				difference--
-			}
-		} else {
-			while (difference !== 0) {
-				GlobalChartState.value.chartTiles.pop()
-				difference++
-			}
-		}
-	},
-	{
-		deep: true
+		SaveSiteOptions()
```

Comment:
Potential issue here is chart save may not trigger anymore when re-arranging tiles already dropped onto the chart. Is this covered somewhere else and I'm just missing that?

Reply:
this is just git reducing the size by as little changes needed. this is a brand new watcher that does different